ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.14.yaml

```diff
@@ -39,6 +39,11 @@ releases:
       architecture: multi
       channel: fast
       version: "4.14"
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks